tools/osv-scanner/main.go

```diff
@@ -252,5 +299,22 @@ func main() {
 		return
 	}
 
-	printResults(query, resp)
+	genericPrintResults(query, resp, func(query *osv.Query) bool {
+		return query.Commit == "" && query.Package.PURL == ""
+	}, func(query *osv.Query) []interface{} {
+		return table.Row{query.Package.Ecosystem, query.Package.Name, query.Version}
+	}, table.Row{"Affected Ecosystem", "Affected Package", "Installed Version"})
```

Comment:
Can we just unify the header columns here? 

e.g. we can print somethign like

| Ecosystem | Package | Version | 
| Debian | foo | 1.0.0 |
| GIT | <commit> | ..sha1 hash..  |

Ideally we can unpack the Package URL and present a consistent Ecosystem/Package label for these too. That can come in another PR. In the meantime maybe something like:

| Package URL | pkg:url without version | version |